modules/gis/src/lib/geojson-to-binary.ts

```diff
@@ -58,7 +61,7 @@ function firstPass(features: Feature[]): FirstPassData {
   let polygonRingsCount = 0;
   let polygonFeaturesCount = 0;
   const coordLengths = new Set<number>();
-  const numericPropKeys = {};
+  const numericPropTypes = {};
```

Comment:
My understanding is that this covers all prop types, not just the numeric ones. Name accordingly?

Reply:
Renamed `numericPropTypes` to `propArrayConstructors` and `NumericArrayConstructor` to `PropArrayConstructor`